pkg/varlinkapi/system.go

```diff
@@ -15,22 +16,14 @@ func (i *LibpodAPI) GetVersion(call iopodman.VarlinkCall) error {
 		return err
 	}
 
-	return call.ReplyGetVersion(iopodman.Version{
-		Remote_api_version: versionInfo.RemoteAPIVersion,
-		Version:            versionInfo.Version,
-		Go_version:         versionInfo.GoVersion,
-		Git_commit:         versionInfo.GitCommit,
-		Built:              versionInfo.Built,
-		Os_arch:            versionInfo.OsArch,
-	})
-}
-
-// Ping returns a simple string "OK" response for clients to make sure
-// the service is working.
-func (i *LibpodAPI) Ping(call iopodman.VarlinkCall) error {
```

Comment:
@baude Any reason to keep this? I think it might be useful for a liveliness check

Reply:
yes, do not remove it; we often use it to test that the connection is working.

Reply:
Hm, I don't feel super strong about it, but it seems weird to me to have a debug call like this in the API. Could you instead use `GetVersion()` for testing that the connection is working?